ckanext/datastore/blueprint.py

```diff
@@ -44,6 +44,40 @@
 
 datastore = Blueprint(u'datastore', __name__)
 
+# (canada fork only): exclude _id field from Blueprint dump
+from ckan.plugins.toolkit import missing, StopOnError
+from flask import has_request_context
+from six import text_type
+def exclude_id_from_ds_dump(key, data, errors, context):
+    """
+    Always set the list of fields to dump from the DataStore. Excluding to _id column.
+
+    This validator is only used in the dump_schema.
+    """
+    value = data.get(key)
+
+    if not has_request_context() and not hasattr(request, 'view_args') and 'resource_id' not in request.view_args:
+        # treat as ignore_missing
+        data.pop(key, None)
```

Comment:
`ignore_missing` does
```
if value is missing or value is None:
    data.pop(key, None)
```
Did you mean `treat as ignore`?

Reply:
Comment updated.